engine/data/score-data-api/pom.xml

```diff
@@ -25,17 +25,17 @@
 
 		<dependency>
 			<groupId>commons-lang</groupId>
-			<artifactId>commons-lang</artifactId>
+			<artifactId>commons-lang</artifactId>
 		</dependency>
-
-		<dependency>
-			<groupId>org.hibernate.javax.persistence</groupId>
-			<artifactId>hibernate-jpa-2.0-api</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>org.hibernate</groupId>
-			<artifactId>hibernate-core</artifactId>
+		
+		<dependency>
+			<groupId>org.eclipse.persistence</groupId>

```

Comment:
As of Spring IO Platform 1.1, the dependency management for org.hibernate.javax.persistence:hibernate-jpa-2.0-api is deprecated due to the use of a version number in its artifact ID. org.eclipse.persistence:javax.persistence is its preferred replacement.